docs/getting-started/installation/mac.md

```diff
@@ -95,6 +95,9 @@ You can also manage your NPM with [asdf](https://nodecli.com/nodejs-asdf).
    - e.g. with SSH: `git clone git@github.com:<your-username>/forem.git`
 
 3. Install bundler with `gem install bundler`
+
+   - If you are using a Mac with an M1 CPU using Ruby compiled for ARM, you may need to force the platform that Bundler uses to `ruby` in order to build some of the C extensions used by Forem's dependencies. To do that, you can either set `BUNDLE_FORCE_RUBY_PLATFORM=true` in the shell environment or run `bundle config set force_ruby_platform true` to set that in Bundler globally.
```

Comment:
Great catch Jamie. Glad we are adding this info to the docs. 

For readability, I have a few suggestions. I would try to break up this section a little so it is not so wordy and the code to force the platform to use ruby is easy to grab. Let me know what you think of this example of how I would write this section: 

**Note:** If you are using a Mac with an M1 CPU that uses Ruby compiled for ARM, you may need to force the platform that Bundler uses to `ruby`. This will allow you to build the C extensions used by Forem's dependencies. 
To force the platform to use `ruby`, you can either: 
- set `BUNDLE_FORCE_RUBY_PLATFORM=true` in the shell environment 

or 
- run `bundle config set force_ruby_platform true` 
to set `ruby` in Bundler globally

Reply:
@cmgorton I wondered about that, too. In hindsight, I should've asked for this kind of feedback in the PR description. 😄 Either way, I appreciate that you gave it and I'll incorporate it now!